### PR TITLE
some suggestions and fixes to grammar

### PR DESCRIPTION
**Pull request by dusual**
Adding some suggestions for text changes and grammar

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -167,7 +167,7 @@ <h3 class="main-title3">Remaining</h3>
                     <div class="event-info-col">
                         <div class="event-info-ico"><span class="fa fa-microphone"></span></div>
                         <h3 class="main-title3">Speakers</h3>
-                        <p><strong>28</strong> Professional Speakers</p>
+                        <p><strong>28</strong>Speakers</p>
```
Comment (dusual) [inline index.html:170]: I am not too sure about using the term "Professional Speakers". I think we are trying to express they are python professional and the term does not seem to be conveying the same sentiment

File: index.html [hunk 2]
```diff
@@ -202,9 +202,9 @@ <h1 class="title3 title-border"><i class="fa fa-dot-circle-o"></i> About the con
                             community</a>. We hope to attract the best Python
                         programmers from across the country and abroad. The
                         main aim of the conference is to bring together
-                        established companies and startups, experts, beginners
-                        and global entrepreneurs under one roof. We will be
-                        inviting experts from various fields to showcase how
+                        startups and established companies, beginners
```
Comment (dusual) [inline index.html:205]: Not sure about this.

File: index.html [hunk 3]
```diff
@@ -314,7 +314,8 @@ <h3 class="schedule-item-title">Welcome Ceremony & Registration</h3>
                                                                 3) inaugurate
                                                                 the conference
                                                                 and say a few
-                                                                words about the conference and its impact on society. Registration would also be going on.</p>
+                                                                words about the conference and its impact on society.
+                                                                We shall parallely be registering the participants during this time.</p>
```
Comment (dusual) [inline index.html:318]: Earlier sentence just seemed to be plain

File: index.html (hunk 1 above)
Comment (theskumar) [inline index.html:170]: +1

File: index.html (hunk 3 above)
Comment (theskumar) [inline index.html:318]: +1

**Comment by theskumar** [comment]
Things look pretty good here. Can you resolve the merge conflicts and we are good to merge.

**Comment by dusual** [comment]
@theskumar done

**Comment by theskumar** [comment]
:heart_decoration: merging.

**Pull request merged**